### PR TITLE
Bump NLTK

### PR DESCRIPTION
**Pull request by ivaquero**

### CODE REVIEW

File: docsets/NLTK/docset.json [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
\ No newline at end of file
+    ],
+    "specific_versions": [
+        {
+            "version": "3.8.1",
+            "archive": "versions/3.8.1/NLTK.tgz"
+        },
+        {
+            "version": "3.0.0",
+            "archive": "versions/3.0.0/NLTK.tgz"
```
Comment (Kapeli) [inline docsets/NLTK/docset.json:21]: Moving the `txt` file does nothing. You need to add this actual archive, or remove the specific version.

**Comment by Kapeli** [comment]
You need to add the actual archive at this path: `versions/3.0.0/NLTK.tgz`, or remove the specific version.

**Comment by ivaquero** [comment]
I will handle this tomorrow.

**Comment by Kapeli** [comment]
Thank you!

**Pull request merged**